src/test/java/uk/gov/pay/adminusers/infra/AppWithPostgresExtension.java

```diff
@@ -121,4 +127,13 @@ private void restoreDropwizardsLogging() {
                 app.getApplication().getName());
     }
 
+    private ConfigOverride[] overrideUrlsConfig(ConfigOverride[] configOverrides) {
+        List<ConfigOverride> newConfigOverride = newArrayList(configOverrides);
+        newConfigOverride.add(config("ledgerBaseURL", "http://localhost:" + wireMockPort));
```

Comment:
Why can't this line be added to the [newConfigOverrides](https://github.com/alphagov/pay-adminusers/pull/2233/files#diff-c7c1b3051e99df0957b92398a358255da4c39a6e78053b5d9e17dd7d58e3857fR56) directly?